<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaterTop</class>
 <widget class="QWidget" name="WaterTop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WaterTop
{
background-color: rgb(14, 26, 50);
}
#widget
{
background-color: rgb(14, 26, 50);
}
QWidget
{
border:0px
}
QLabel
{
color: rgb(122, 175, 227);
font: 12pt &quot;楷体&quot;;
border:0px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>330</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>999</width>
       <height>9999</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="TempWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#TempWidget
{

}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="TempIcon">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/ptr/Temp.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TempName">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>温度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TempValue">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt ;
font-weight:bold;
</string>
           </property>
           <property name="text">
            <string>24.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TempUnit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>℃</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="HumidityWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="HumidityIcon">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/ptr/humidity.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="HumidityName">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>湿度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="HumidityValue">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt ;
font-weight:bold;
</string>
           </property>
           <property name="text">
            <string>68.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="HumidityUnit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="PressureWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="PressureIcon">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/ptr/pressure.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PressureName">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>气压</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PressureValue">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt ;
font-weight:bold;
</string>
           </property>
           <property name="text">
            <string>1000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PressureUnit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>hPa</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="VocWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="VocIcon">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/ptr/voc.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VocName">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>VOC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VocValue">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt ;
font-weight:bold;
</string>
           </property>
           <property name="text">
            <string>11.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VocUnit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>ppm</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="LightWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="LightIcon">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/ptr/Light.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LightName">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>光照强度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LightValue">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt ;
font-weight:bold;
</string>
           </property>
           <property name="text">
            <string>800</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LightUnit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>999</width>
             <height>999</height>
            </size>
           </property>
           <property name="text">
            <string>lux</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
